modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -85,6 +85,37 @@ Note: If the payload is used as part of the message log, do not configure the Li
 |After Calling API |Set to cause your logging policy to log messages after the API endpoint is called, taking into account the order in which this policy is applied |Optional
 |===
 
+== Example of Logging Policy from the UI
+
+To better understand how to use the Message Logging policy, consider an example "Les Vetments" company that is in the business of retail clothing and more. To run their e-commerce portal successfully, Les Vetments needs to audit every event and transaction made on their website.
```

Comment:
english is not my strength, but maybe it's consider as **an** example?

Reply:
Leaving it to my editor. This is how she suggests we write though.